hypertrace-graphql-labels-schema/src/main/java/org/hypertrace/graphql/label/dao/LabelRequestConverter.java

```diff
@@ -8,15 +9,27 @@
 
 public class LabelRequestConverter {
   CreateLabelRequest convertCreationRequest(LabelCreateRequest creationRequest) {
-    return CreateLabelRequest.newBuilder()
-        .setData(LabelData.newBuilder().setKey(creationRequest.label().key()).build())
-        .build();
+    LabelData.Builder dataBuilder = LabelData.newBuilder().setKey(creationRequest.label().key());
+    if (Objects.nonNull(creationRequest.label().color())) {
+      dataBuilder.setColor(Objects.requireNonNull(creationRequest.label().color()));
```

Comment:
once a null check is done, we can directly use the value. Isn't it?

Reply:
Should be able to. Style nit, but could also rewrite this as 

```java
Optional.ofNullable(creationRequest.label().color())
  .ifPresent(dataBuilder::setColor);
```